docs/code/Flakes/Tutorials/Hacking.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 03-Create a Development Environment
+date: 2021-02-09
+---
```

Comment:
It's possible to order documents in their folder by adding:
```suggestion
---
title: 03-Create a Development Environment
date: 2021-02-09
order: 3
---
```

Reply:
Addressed in a7bd2f9